OracleEDQ/README.md

```diff
@@ -0,0 +1,120 @@
+Oracle Enterprise Data Quality on Docker
+========================================
+Sample Docker configurations to facilitate installation, configuration, and environment setup for DevOps users. This project includes quick start [dockerfiles](dockerfiles/) for Enterprise Data Quality 12.2.1.3.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle FMW Infrastructure 12.2.1.3.0.
+
+For more information about Oracle Enterprise Data Quality please see the [Oracle Enterprise Data Quality 12.2.1.3.0 Online Documentation]( http://www.oracle.com/technetwork/middleware/oedq/documentation/index.html).
+
+The certification of Oracle Enterprise Data Quality on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+
+For pre-built images containing Oracle software, please check the [Oracle Container Registry](https://container-registry.oracle.com).
+
+## Database prerequisite
+
+You need to have a running Oracle Database, either in a Docker container or on a host. 
+The database connection details are required for creating midtier schemas for use by the EDQ domain.  The schemas are created automatically when the EDQ container is started.
+ 
+If using a 12.1.0.2 CDB/PDB database, ensure PDB is used when creating the schemas. CDB is not supported.
+
+An Oracle Database 12.1.0.2 container can be created from [OracleDatabase](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.1.0.2). 
+Follow the instructions to create a 12.1.0.2-based Enterprise Edition database.
+
+Default installs of Oracle Database 12.2 are not supported - please see [Known Issues](#known-issues).
+
+## Oracle Enterprise Data Quality Docker Image Creation
+
+To build a EDQ image you start by building Oracle JDK and Oracle FMW Infrastructure images.
+
+### Building the Oracle JDK (Server JRE) Image
```

Comment:
This should be "Oracle Java 8 (Server JRE) Image"

Reply:
Fixed, changed to 
### Building the Oracle Java 8(Server JRE) Image
o build a EDQ image you start by building Oracle Java 8(Server JRE) and Oracle FMW Infrastructure images.

### Building the Oracle Java 8(Server JRE) Image

Download the Oracle Server JRE binary into folder `OracleJava/java-8` and build the image: